### PR TITLE
add ability to link to a location

### PR DESCRIPTION
**Pull request by martinamps**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -11,8 +11,7 @@
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/apple-touch-icon.png" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <link href="https://api.mapbox.com/mapbox-gl-js/v1.7.0/mapbox-gl.css" rel="stylesheet" />
```
Comment (jorge-caballero) [inline public/index.html:14]: nice catch

**Review by jorge-caballero — COMMENTED** [review]
i'd like to ask @afahy to take a look as well. the implementation here may be helpful to consider for the project he's working on right now

File: src/App.tsx [hunk 2]
```diff
@@ -58,19 +59,22 @@ const defaultFilters: SearchFilterType = {
...
-  timestamp: number;
+class LatLng {
+  lat: number;
+  lng: number;
+
+  constructor(lat: number, lng: number) {
+    this.lat = lat;
+    this.lng = lng;
+  }
 }
```
Comment (afahy) [inline src/App.tsx]: This seems unnecessary. Line 116 can just do `{ lat, lng }` instead and you get the same result

File: src/App.tsx [hunk 3]
```diff
@@ -97,6 +102,46 @@ const App = () => {
...
+      let currentPath = document.location.pathname.substr(1);
+      let uriPlace = (selectedPlace as any);
+      if (uriRegex.test(currentPath)
+        && (!selectedPlace || (selectedPlace as any).location_id !== currentPath)) {
+  
+        fetchLocation(currentPath).then(newPlace => {
+          if (isFirstLoad ||
+            (uriPlace && uriPlace.location_id === newPlace.location_id)) {
+            setSelectedPlace(newPlace);
+            globalMap.panTo(new LatLng(newPlace.location_latitude, newPlace.location_longitude));
```
Comment (afahy) [inline src/App.tsx]:
```suggestion
            globalMap.panTo({ lat: newPlace.location_latitude, lng: newPlace.location_longitude });
```

File: src/App.tsx [hunk 4]
```diff
@@ -97,6 +102,46 @@ const App = () => {
     setShowAppointmentFlow(true);
   }
 
+  useEffect(() => {
+    function triggerRouter() {
```
Comment (afahy) [inline src/App.tsx:97]: I feel like we actually want to use a routing library longer-term, but if this works 👍

**Review by afahy — APPROVED** [review]

**Review by jorge-caballero — APPROVED** [review]

**Pull request merged**